go.mod

```
@@ -1,10 +1,12 @@
 module github.com/salsadigitalauorg/shipshape
 
-go 1.17
+go 1.21
```

Another one here for pinning the go version:
```suggestion
go 1.20
```